docs/cli-usage.md

```diff
@@ -43,7 +43,7 @@ djinni \
 | `--help` | Print help  |
 | `--version` | Print version |
 | `--idl <in-file>` | The IDL file with the type definitions, typically with extension `.djinni`. |
-| `--idl-include-path <path> ...` | An include path to search for Djinni `@import` directives. Can specify multiple paths. |
+| `--idl-include-path <path> ...` | An include path to search for Djinni `@import` and `@extern` directives. Can specify multiple paths. |
```

Comment:
As you can see [here](https://github.com/cross-language-cpp/djinni-generator/blob/main/src/main/scala/djinni/parser.scala#L46), the include path seems to affect both `@import` and `@extern`.

Reply:
good catch!